packages/legend-query/src/components/__tests__/QueryBuilder_FetchStructure.test.tsx

```diff
@@ -487,3 +489,591 @@ test(
   },
   // TODO: add more test when we rework the graph fetch tree
 );
+
+test(
```

Comment:
Can we move these test to a separate file, something like `QueryBuilder_Milestoning.test.tsx` and can we also do `Business Temporal` and `Processing Temporal` in the test name instead of `businesstemporal`? Thanks!